packages/bsky/src/lexicon/lexicons.ts

```diff
@@ -980,6 +983,44 @@ export const schemaDict = {
       },
     },
   },
+  ComAtprotoAdminRebaseRepo: {
```

Comment:
Some of these changes, along with other bsky changes are from codegen, unrelated to the PR itself.

Reply:
Yeah, when running codegen in bsky, you have to stage only the relevant changes before commiting, if you're using git from the terminal you can use `git add -p` if not, I think vscode allows you to stage individual hunks

Reply:
thanks but I was told to keep bsky in sync with the lexicons, at least for the time being, so I think these are safe to commit.